tensorflow/python/training/adam.py

```diff
@@ -29,7 +29,7 @@
 
 
 class AdamOptimizer(optimizer.Optimizer):
-  """Optimizer that implements the Adam algorithm.
+  """Optimizer that implements the Adam algorithm (http://arxiv.org/abs/1412.6980).
```

Comment:
Done! Pointing to v7 and removed the duplicate reference.

On Sat, Jan 2, 2016 at 7:39 PM, rafaljozefowicz <notifications@github.com>
wrote:

> In tensorflow/python/training/adam.py
> <https://github.com/tensorflow/tensorflow/pull/670#discussion_r48689777>:
>
> > @@ -29,7 +29,7 @@
> >
> >
> >  class AdamOptimizer(optimizer.Optimizer):
> > -  """Optimizer that implements the Adam algorithm.
> > +  """Optimizer that implements the Adam algorithm (http://arxiv.org/abs/1412.6980).
>
> I think what you have looks good. Can you remove that other reference and
> maybe point to specific version of the paper (v7)? The formulas were
> updated from previous versions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/670/files#r48689777>.
>